README.md

```diff
@@ -276,16 +273,6 @@ result = mq.index("my-first-index").search('marco polo', search_method=marqo.Sea
 
 ```
 
-### Search specific fields
```

Comment:
Since `searchable_attributes` hasn't been supported yet with V2, I removed the whole section of `Search specific fields`